hasura.planx.uk/migrations/1700845956731_alter_table_public_analytics_logs_add_column_allow_list_answers/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."analytics_logs" add column "allow_list_answers" jsonb
+ null default '[]'::jsonb;
```

Comment:
I wonder if there are any benefits/clarity of adding this column as [an array type](https://www.postgresql.org/docs/current/arrays.html) (eg `text[]`) rather than `jsonb`? 

An array feels more representative of the actual expected data structure to me, but I'd be curious if there's any notable Metabase pros/cons to working with one type over the other?

Reply:
My understanding of this is that the `allow_list_answers` column is  `jsonb` but it defaults to `[]` if a value isn't provided. 

My thinking was that at the moment we expect this to be an array but it being `jsonb` gives us flexibility.

I was basing this on https://github.com/theopensystemslab/planx-new/pull/2454/files#diff-600c74a4ab8a37ce56adffbba43f1443538b77fcc91d4b9db7f344eb0066c167 although I've maybe misunderstood?

I'm not sure if this really answers your question 🤔

Reply:
Mimicing behavior or recently added input errors column works for me, thanks for pointing that out!